pushapkscript/script.py

```diff
@@ -17,15 +17,17 @@
 from pushapkscript.apk import sort_and_check_apks_per_architectures
 from pushapkscript.task import validate_task_schema, extract_channel
 from pushapkscript.utils import load_json
-from pushapkscript.googleplay import publish_to_googleplay
+from pushapkscript.googleplay import publish_to_googleplay, is_allowed_to_push_to_google_play, will_commit_transaction
 
 
 log = logging.getLogger(__name__)
 
 
 async def async_main(context):
     context.task = scriptworker.client.get_task(context.config)
-    log.info('Validating task')
+    _print_warning_forewords(context)
```

Comment:
tiny nit: using the `print` within the name function is slightly deceiving. Should we `s/_print_warning_forewords`/log_warning_forewords` instead since all printing are logs calls anyway?

Reply:
Good call. Renamed